server/plugin/plugin.go

```diff
@@ -250,6 +274,151 @@ func (p *Plugin) OnDeactivate() error {
 	return nil
 }
 
+func (p *Plugin) getPostPropsForReaction(reaction *model.Reaction) (orgRepo []string, id float64, objectType string, ok bool) {
+	post, err := p.client.Post.GetPost(reaction.PostId)
+	if err != nil {
+		p.API.LogDebug("Error fetching post for reaction", "error", err.Error())
+		return orgRepo, id, objectType, false
+	}
```

Comment:
This is costly. For most of the reactions posted, an RPC request is made to fetch the post. It's a bummer `ReactionHasBeenAdded` doesn't provide the post.

Anyway, I don't see a way to get around the RPC call without changing the hook.